db/migrate/20160721103649_add_industry_to_profile.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndustryToProfile < ActiveRecord::Migration
+  def change
+    add_column :profiles, :industry, :string
```

Comment:
1) Checkout the branch where you need to put it back in (I'll call it branch A)
2) `git checkout customer_username db/migrate/20160721103649_add_industry_to_profile.rb`
3) `git add` and `git commit` in Branch A
4) Checkout customer_username again, `git rm` this file and then `git push`
:+1: